backend/src/main/java/se/chalmers/dat265/group1/WebApi.java

```diff
@@ -94,6 +109,21 @@ private static void setupDigitalPartsInterface() {
         put(DIGITALPARTS_URL + DIGITALPART_ID_URL, (request, response) -> dpi.updateDigitalPart(gson.fromJson(request.body(), DigitalPart.class)), gson::toJson);
     }
 
+    private static void setupPhysicalInterface() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "setupPhysicalInterface" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)